docs/pages/application-access/getting-started.mdx

```diff
@@ -67,45 +84,60 @@ $ tctl tokens add \
     --app-uri=http://localhost:3000
 ```
 
-<Tabs>
-<TabItem label="Docker">
+## Step 2/5. Start Grafana
 
-## Step 2/4. Start Grafana
+You can start Grafana in a Docker container or Kubernetes cluster by running the appropriate steps.
+For more information, click **Docker** or **Kubernetes cluster**.
 
-We've picked Grafana for this tutorial since it's very easy to run with zero
-configuration required. If you have another web application you'd like to
-expose, skip over to **Step 3**.
+<Tabs>
+<TabItem label="Docker">
 
-Grafana can be launched in a Docker container with a single command:
+You can start Grafana in a Docker container by running the following command:
 
 ```code
 $ docker run -d -p 3000:3000 grafana/grafana
 ```
 
-Next, we need to edit `/etc/grafana/grafana.ini` in the container. Under
-`[server]`, uncomment the `domain` key and set the value to your Teleport
-cluster domain, (e.g., `teleport.example.com`). Since there is no text editor
-installed in the container, we can do this with `sed`:
+After running this command, edit the `/etc/grafana/grafana.ini` file in the container. 
+Under `[server]`, uncomment the `domain` key and set the value to your Teleport
+cluster domain, (for example, `teleport.example.com`). 
+Because the container doesn't have a text editor installed, you can use the `sed` command to edit the file. 
+For example:
 
 ```code
 $ docker exec -u 0 grafana sed -i 's/;domain = localhost/domain = teleport.example.com/g' /etc/grafana/grafana.ini
 ```
 
 Replace `teleport.example.com` with your Teleport cluster domain.
+</TabItem>
+<TabItem label="Kubernetes cluster">
 
-## Step 3/4. Install and configure Teleport
+If you install Grafana with the following `helm` instructions, it is available at `http://example-grafana.example-grafana.svc.cluster.local`
+within the Kubernetes cluster.
+
+```code
+$ helm repo add grafana https://grafana.github.io/helm-charts
+$ helm repo update
+$ helm install example-grafana grafana/grafana \
+    --create-namespace \
+    --namespace example-grafana
+```
+</TabItem>
+</Tabs>
 
-(!docs/pages/includes/permission-warning.mdx!)
+## Step 3/5. Install and configure Teleport
 
-### Configure and Start Teleport
+Install Teleport on the host where you will run the Teleport Application Service.
 
-Install Teleport on the host where you will run the Teleport Application Service. See our [Installation](../installation.mdx) page for options besides Linux servers.
+<Tabs>
+<TabItem label="Docker">
+See the [Installation](../installation.mdx) page for options besides Linux servers.
 
 (!docs/pages/includes/install-linux.mdx!)
 
 Create the Teleport Application Service configuration at `/etc/teleport.yaml`.
 Replace <Var name="teleport.example.com" /> with your Teleport Proxy Service
-address (`teleport.example.com`) or Teleport Cloud tenant (e.g. `mytenant.teleport.sh`).
+address or Teleport Cloud tenant (for example, `mytenant.teleport.sh`).
```

Comment:
Here and below, why remove the example of a self-hosted proxy address?

Reply:
It seemed clutter-y to keep repeating `teleport.example.com` everywhere and I'm not sure what the rationale is for using different terms for the `teleport.example.com` address (Teleport cluster name, Teleport Proxy Service address, domain, URL).